.github/scripts/fetch_external_sources.sh

```diff
@@ -1,9 +1,28 @@
 #!/usr/bin/env bash
 # Script that tries to automatically fetch further documentation from other
 # github repositories, where it simply assumes that they are available under the
-# same name. Runs through our ususal suspects of github organizations while
+# same name. Runs through our usual suspects of github organizations while
 # trying to do so
 
+# Initialize force option
+FORCE=false
+
+# Parse command line arguments
+for arg in "$@"; do
+  case $arg in
+    --force|-f) FORCE=true ;;
+    --help|-h)
+      echo "Usage: $0 [--force|-f] [--help|-h]"
+      echo ""
+      echo "Options:"
+      echo "  --force, -f    Force fetching files even if they already exist"
+      echo "  --help, -h     Display this help message"
+      exit 0
+      ;;
+    *) echo "Unknown parameter passed: $arg"; exit 1 ;;
```

Comment:
It would be nice if this displayed the usage (without verbose help)

Reply:
Good idea